components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -46,8 +47,13 @@ - (void)drawRect:(CGRect)rect {
 @end
 
 @interface TBVCSampleViewController ()
+
 @property(nonatomic) MDCAppBarViewController *appBarViewController;
 @property(nonatomic) UILabel *titleLabel;
+@property(nonatomic) CGRect buttonFrame;
```

Comment:
Please document that this is the "target" frame desired by the calling code. Reading the property, I thought it was going to be just a copy of `self.button.frame`.

Reply:
Good point! Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.